src/System.Private.CoreLib/src/System/Runtime/InteropServices/InteropExtensions.cs

```diff
@@ -357,12 +357,7 @@ public static uint RuntimeHandleCompareExchangeVariableType(IntPtr handle, uint
         {
             return RuntimeImports.RhHandleCompareExchangeVariableType(handle, oldType, newType);
         }
-
-        public static void SetExceptionErrorCode(Exception exception, int hr)
```

Comment:
You may want to keep this around, under `#if PROJECTN // TODO: delete`. ProjectN integration will break without it.

Reply:
Add back these functions and will remove them in ProjectN